frontend/src/containers/Ledger.js

```diff
@@ -157,9 +159,19 @@ export class Ledger extends Component {
       fetchUsers,
       fetchPendingExpenses,
       fetchTransactions,
-      loadData
+      loadData,
+      params
     } = this.props;
 
+    switch (params.action) {
+      case 'approve':
+        this.approveExp(params.expenseid);
+        break;
+      case 'reject':
+        this.rejectExp(params.expenseid);
+        break;
+    }
+
```

Comment:
It's using the same logic then when the user clicks on the "approve" button.
The user will see the expense with the label "approved" in the list of pending expenses to be paid at the top.